.github/workflows/clang-format-check.yml

```diff
@@ -0,0 +1,12 @@
+name: clang-format Check
```

Comment:
I don't really like the capitalization I would prefer "clang-format check" or `ClangFormat Check`

Reply:
OK. Let me change and see if we can do without the `exclude`.